README.md

```diff
@@ -29,43 +31,43 @@ rfid.on('ready', function (version) {
 });
 ```
 
-###Methods
+### Methods
 
-&#x20;<a href="#api-rfid-setPollPeriod-pollPeriod-callback-err" name="api-rfid-setPollPeriod-pollPeriod-callback-err">#</a> rfid<b>.setPollPeriod</b>( pollPeriod, callback(err) )  
- Set the time in milliseconds between each check for an RFID device. The poll period is 500ms by default.
+* &#x20;<a href="#api-rfid-setPollPeriod-pollPeriod-callback-err" name="api-rfid-setPollPeriod-pollPeriod-callback-err"></a> rfid<b>.setPollPeriod</b>( pollPeriod, callback(err) )  
```

Comment:
These are actually done in this way intentionally– it lets us link to specific methods in the API e.g. https://github.com/tessel/rfid-pn532#api-rfid-on-read-callback-data